src/suricata.c

```diff
@@ -2860,28 +2859,16 @@ static void SuricataMainLoop(SCInstance *suri)
         }
 
         if (sigusr2_count > 0) {
```

Comment:
Suggest `if (sigusr2_count)` with `unlikely` here (and for the other conditionals in this loop).

Reply:
Not sure if this is important, as this is a loop that sleeps a lot. Btw this is also not part of the changeset that is being proposed, so a 'optimization' ticket would be the proper place normally.

Reply:
Given that, then I agree ... using `unlikely, likely` are not necessary.